sounddevice.py

```diff
@@ -2289,13 +2289,40 @@ def reset(self):
         vars(self).clear()
         self.__init__()
 
+
 if not hasattr(_ffi, 'I_AM_FAKE'):
     # This object shadows the 'default' class, except when building the docs.
     default = default()
 
 
 class PortAudioError(Exception):
-    """This exception will be raised on PortAudio errors."""
+    """This exception will be raised on PortAudio errors.
+
+    Attributes
+    ----------
+    args
+        A variable length tuple containing the following elements when
+        available:
+
+        1) The user error message
```

Comment:
Well as we currently use it, this is most of the times a combination of a message given by the `sounddevice` module and a message given by PortAudio. Sometimes it's only the former, sometimes it's only the latter.

I think we can stay vague and just write "The error message" or "A string describing the error" or something like that.